c++/src/kj/async-io-unix.c++

```diff
@@ -216,7 +216,7 @@ public:
       return downcast<AsyncStreamFd>(*stream).fd;
     };
     auto promise = writeInternal(data, moreData, fds);
-    return promise.attach(kj::mv(fds));
+    return promise.attach(kj::mv(fds), kj::mv(streams));
```

Comment:
BTW, I must have read this line of code 5 times in the last 2 days and completely missed this.

Reply:
I missed it a few times too. Didn't dawn on me until I read the calling code and noticed that ownership of the streams array was being passed in.